ace_studio_streamlit/components/generation_wizard.py

```
@@ -0,0 +1,370 @@
+            "**⚙️ Settings → Models**."
+        )
+        if st.button(
+            "🚀 Load DiT Model Now",
+            key="quick_init_dit",
+            type="primary",
+        ):
+            _quick_init_dit()
+            return
+
+    # ------------------------------------------------------------------
+    # Section 1 – Inspiration & Vibe
```

Ruff (0.15.2)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)